src/bitcoind_rpc.rs

```diff
@@ -2,13 +2,15 @@
 
 use ::bitcoin::{hashes::hex::FromHex, Address, Amount, Network, Transaction, Txid};
 use bitcoincore_rpc_json::{FinalizePsbtResult, GetAddressInfoResult};
-use jsonrpc_client::{JsonRpcError, ResponsePayload, SendRequest};
+use jsonrpc_client::{ResponsePayload, SendRequest};
 use reqwest::Url;
 use serde::de::DeserializeOwned;
 use serde::Deserialize;
 use std::collections::HashMap;
 
+use self::jsonrpc_client::JsonRpcError;
```

Comment:
These two changes seem to be unnecessary :)

Reply:
In fact, line 11 is a bit confusing because you are re-exporting from a module that isn't even defined yet.

Reply:
Oh, yeah, I think the `JsonRpcError` should not have been removed above. Fixing it.